code/api/src/modules/user/resolvers.js

```diff
@@ -74,7 +72,8 @@ export async function update(parentValue, { name, email, description, address, i
       },
       {where: {id}}
     )
-    return auth.user
+    let user = models.User.findOne({ where: { id }})
```

Comment:
Hopefully this fixes our bug where a user needs to log out/log in to view updates to their profile.